doc/devguide/codebase/code-style.rst

```diff
@@ -275,6 +673,40 @@ Goto statements should be used with care. Generally, we use it primarily for err
         return NULL;
     }
 
+Nested goto labels are indented.
+
+.. code-block:: c
+
+    int goto_style_nested()
+    {
+        if (foo()) {
+        label1:
+            bar();
+        }
+
+    label2:
+        return 1;
+    }
+
+TODO - REMOVEME: This is only configurable to left-most as of clang 10 so we
+could just leave the "nested goto" out to "not overload things".
```

Comment:
Or require clang 10. As @victorjulien mentioned and I've tested with. It's available on ubuntu 18.04. Requires fedora 32, afaik.

Reply:
I don't have a strong enough preference to force clang-10 on ppl

Reply:
Will remove TODO in v5.

Reply:
Rephrased to "Put goto labels at brace level"

Reply:
Sticking to clang 9 for now in this case